fmt/src/formatter.rs

```diff
@@ -3333,6 +3333,11 @@ impl<'a, W: Write> Visitor for Formatter<'a, W> {
                 self.visit_list("", idents, Some(loc.start()), Some(loc.end()), false)?;
                 None
             }
+            VariableAttribute::StorageType(s) => match s {
```

Comment:
is there any reason the order of the matching should match the [order of the enum?](https://github.com/stellar/rs-soroban-env/blob/cf469d2d4be82f83c24b75aeb8a133cfb82a23cd/soroban-env-common/src/storage_type.rs#L12)